allennlp/tests/modules/drop_connect_test.py

```diff
@@ -47,6 +48,22 @@ def test_lstm_outputs(self):
         output_b, _ = dropped_lstm(input_tensor)
         assert torch.allclose(output_a, output_b)
 
+    @unittest.skipIf(not torch.cuda.is_available(), reason="No CUDA device registered.")
+    @flaky(max_runs=10, min_passes=1)
```

Comment:
Why run this test multiple times?

Reply:
My bad. I tried to parametrize the original `test_lstm_outputs()` and then realized that it probably won't work well with `@flaky` and won't make sense anyway.
Will remove the `@flaky`.